### PR TITLE
Add shelter and event dropdown to EVASS template

### PR DESCRIPTION
**Pull request by svineet**
Melange task: http://www.google-melange.com/gci/task/view/google/gci2014/5796656591142912
ToDo in code: https://github.com/flavour/eden/blob/ebe38ab2373ba486f746d8e96e04bc228d8bd611/private/templates/EVASS/controllers.py#L195com/flavour/eden/blob/4ae1744b8a9b3e27c86fd2b36e2e06d895

### CODE REVIEW

File: private/templates/EVASS/controllers.py [hunk 1]
```diff
@@ -192,8 +198,21 @@ def shelter_list(self):
                       shelter_list
                       )
         else:
-            # @todo: check permission and provide an "Add Shelter" button
-            #        if not shelters are yet registered
+            if current.auth.s3_has_permission("write", current.db.cr_shelter):
```
Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: "create"(CRUD)

File: private/templates/EVASS/controllers.py [hunk 2]
```diff
@@ -224,19 +250,24 @@ def event_list(self):
                       event_list
                       )
         else:
-            # @todo: check permission and provide an "Add Event" button
-            #        if not events are yet registered?
+            if current.auth.s3_has_permission("write", current.db.event_event):
```
Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: "create"(CRUD)

File: private/templates/EVASS/controllers.py [hunk 3]
```diff
@@ -192,8 +198,21 @@ def shelter_list(self):
                       shelter_list
                       )
         else:
-            # @todo: check permission and provide an "Add Shelter" button
-            #        if not shelters are yet registered
+            if current.auth.s3_has_permission("write", current.db.cr_shelter):
+                shelter_list.append(
```
Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: If the user has the permission, then just show him a button to create a shelter. If he/she doesn't, then you could keep a button for the module page. I think it's better if it's just a button and not a drop-down.
Same for events

File: private/templates/EVASS/controllers.py [hunk 4]
```diff
@@ -209,6 +228,13 @@ def event_list(self):
                         _class = "f-dropdown",
                         data = {"dropdown-content": ""})
         rows = data["rows"]
+
+        event_list.append(
```
Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: No need for this ?(Same for shelter)

**Comment by svineet** [comment]
@hitesh96db done the changes

**Comment by svineet** [comment]
@hitesh96db ok now done

File: private/templates/EVASS/controllers.py [hunk 5]
```diff
@@ -192,8 +192,14 @@ def shelter_list(self):
                       shelter_list
                       )
         else:
-            # @todo: check permission and provide an "Add Shelter" button
-            #        if not shelters are yet registered
+            if current.auth.s3_has_permission("create", current.db.cr_shelter):
+                return LI(A(T("Create Shelter"),
+                            _href=URL(c="cr",
+                            f="shelter",
```
Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: Indentation needs to be corrected. Look here
https://github.com/edengci/eden/pull/56/files#diff-8b45f03d431087c74ca9eecb0d6680abL215

**Comment by hitesh96db** [comment]
Minor change else :+1:

File: private/templates/EVASS/controllers.py (hunk 5 above)
Comment (svineet) [inline private/templates/EVASS/controllers.py]: @hitesh96db that's for pep8

Comment (hitesh96db) [inline private/templates/EVASS/controllers.py]: I think it's better if it's corrected :)

**Pull request merged**